js/review-data-types-1.js

```diff
@@ -0,0 +1,81 @@
+// integer variables
+
+var a= 100;
+console.log(a);
+var b=200;
+console.log(b);
+var c=50;
+console.log(c);
+var d=150;
+console.log(d);
+var e=300;
+console.log(e);
+
+// string variables
+
+var a= '100';
+console.log(a);
+var b='200';
+console.log(b);
+var c='50';
+console.log(c);
+var d='150';
+console.log(d);
+var e='300';
+console.log(e);
+
+// boolean variables
+
+var a = true;
+var b = false;
+
+console.log(a);
+console.log(b);
+
+// array variables
+
+var arr1=['pravallika', 'prasad', 'sahasra'];
+var arr2=['hello', 'world'];
+var arr3=['Images','videos','maps'];
```

Comment:
You should follow same naming format/convention for the values. For example, 'Images' should match other items in the list. So it should be 'images'